manifests/database/postgresql.pp

```diff
@@ -56,7 +56,7 @@
 
     exec { 'cpdb':
       path    => '/usr/share/candlepin:/bin',
-      command => "cpdb --create \
+      command => "cpdb --update \
```

Comment:
If you look at the next option passed, we use create with schema-only because the schema has to get created. (https://github.com/candlepin/candlepin/blob/master/server/code/setup/cpdb#L283) which calls a different liquibase XML file:

Schema create: https://github.com/candlepin/candlepin/blob/master/server/code/setup/cpdb#L67
Update: https://github.com/candlepin/candlepin/blob/master/server/code/setup/cpdb#L80

Sounds like maybe we need two exec's one for first time setup and one for migrations there after.

Reply:
I don't understand why the tool doesn't allow `--create --upgrade` in a single run because nothing in the code suggests that would be an issue, but I guess you're right and we have to work around that with 2 execs